add holidays to core standing time

stefansc1 opened the pull request.

jakobgemassmer reviewed (changes requested):
Code in util.py works. Holidays are also written into scenario.json, but not treated equally to full_days in the flex_band. The flex_band ignores the "holidays". Holidays are also not consired, when generating synthetic driving profiles.

- [x] Treat holidays equally to full_days in `generate_schedule.py`.
- [ ] Add holidays to `generate.py` (via generate.cfg as in generate_schedule.cfg), so they are consired when creating syntetic driving profiles (like `no_drive_days`).

stefansc1 commented:
- added holidays to generate and generate.cfg (config-option only)
- using datetime.weekday instead of isoweekday (off-by-one). Sunday is now 6 instead of 7
- renamed full_days to no_drive_days
- updated tests
- updated RTD

jakobgemassmer commented on generate_schedule.py:
```diff
@@ -21,7 +21,7 @@ def generate_flex_band(scenario, gcID, core_standing_time=None):
     :type gcID: string
     :param core_standing_time: core standing time during which flexibility is guaranteed e.g.
-        {"times":[{"start": [22,0], "end":[5,0]}], "full_days":[7]}
+        {"times":[{"start": [22,0], "end":[5,0]}], "full_days":[6]}
```
```suggestion
        {"times":[{"start": [22,0], "end":[5,0]}], "no_drive_days":[6]}
```

jakobgemassmer reviewed (changes requested):
Just a minor name change in the docstrings of generate_schedule.py. Everything else seems solid.
